lib/ruby-debug-ide.rb

```diff
@@ -4,14 +4,16 @@
 require 'thread'
 if RUBY_VERSION < '2.0' || defined?(JRUBY_VERSION)
   require 'ruby-debug-base'
+  Debugger::FRONT_END = "ruby-debug-base"
 else
   require 'debase'
+  Debugger::FRONT_END = "debase"
 end
 
-require 'ruby-debug-ide/version'
-require 'ruby-debug-ide/xml_printer'
-require 'ruby-debug-ide/ide_processor'
-require 'ruby-debug-ide/event_processor'
+require_relative 'ruby-debug-ide/version'
```

Comment:
ruby-debug-ide is somewhat expected to work on 1.8, I believe that require_relative is not available in 1.8